our-work/software/vr-volume-viewer.yml

```diff
@@ -0,0 +1,14 @@
+title: Desktop VR Volume Viewer
+description: A simple viewer for visualizing volumetric datasets. It supports color mapping using transfer functions to map specific values of data to a scalar color. The view camera is fully interactive, and it works on multiple systems (Windows and Linux). It also support multiple displays devices (desktop, VR HMDs, cave-systems).
+languages:
+  - c++
+tags:
+  - vr
+people:
```

Comment:
Should we add Ben ? He ported the shaders from OpenGL 2 to OpenGL 3 and the imgui integration.

Reply:
Ok. understood.